website/src/data/users.js

```diff
@@ -124,6 +124,14 @@ export const Tags = {
 // Add your site to this list
 // prettier-ignore
 const Users = [
+  {
+    title: 'Aide Jeune',
+    description: 'French Discord server that helps young people who have been bullied or feel bad about themselves',
+    preview: require('./showcase/aide_jeune.png'),
+    website: 'https://aidejeune.fr',
+    source: 'https://github.com/AideJeune',
+    tags: ['opensource', 'favorite', 'product'],
```

Comment:
```suggestion
    tags: ['opensource'],
```

It's our responsability to add the favorite tag, and you are not building a product